src/Common/Core/Impl/Security/SecurityUtilities.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+namespace Microsoft.Common.Core.Security {
+    public static class SecurityUtilities {
+        public static IntPtr CreatePasswordBuffer() {
+            return CreateSecureStringBuffer(NativeMethods.CREDUI_MAX_PASSWORD_LENGTH);
+        }
+
+        public static SecureString ToSecureString(this string s) {
```

Comment:
This code was actually taken from where the SS was getting created off the Claims.Password

---
In reply to: [82293962](https://github.com/Microsoft/RTVS/pull/2447#discussion_r82293962) [](ancestors = 82293962)